CMakeLists.txt

```diff
@@ -196,7 +196,7 @@ if (USE_GLOG)
   list(APPEND CAFFE2_INTERFACE_LIBS ${GLOG_LIBRARIES})
 endif()
 if (USE_GFLAGS)
-  list(APPEND CAFFE2_INTERFACE_LIBS ${GFLAGS_LIBRARIES})
+  list(APPEND CAFFE2_INTERFACE_LIBS gflags)
```

Comment:
I expect this will be interpreted as the gflags **CMake target**, not -lgflags, correct?

Reply:
Yep, this is going to be interpreted as the gflags target, which is what public/gflags.cmake does: newer gflags ships with the target definition, for older gflags we construct that target for cmake.